ui/raidboss/data/07-dt/hunts/shaaloani.ts

```diff
@@ -28,16 +28,156 @@ const yeheheOutputStrings = {
   next: Outputs.next,
 } as const;
 
+//     SOUTH
+// -------------
+// OFL OF- OFR |Outer
+// IFL IF- IFR |Inner
+// IBL IB- IBR |Inner
+// OBL OB- OBR |Outer
+//   OF  IF  IB  OB
+//   L-R L-R L-R L-R
+// 0b000_000_000_000
+enum TtokSafeSpots {
+  None = 0,
+  Left = 0b100_100_100_100,
+  Right = 0b001_001_001_001,
+  // ------------------
+  In = 0b000_111_111_000,
+  Out = 0b111_000_000_111,
+  Front = 0b111_111_000_000,
+  Back = 0b00_000_111_111,
+  FR_BL = 0b001_001_100_100,
+  FL_BR = 0b100_100_001_001,
+  InRight = TtokSafeSpots.In & TtokSafeSpots.Right,
+  InLeft = TtokSafeSpots.In & TtokSafeSpots.Left,
+}
+
+const ttokrroneTempestSandspoutOutputStrings = {
+  unknown: Outputs.unknown,
+  in: Outputs.getUnder,
+  out: Outputs.outOfMelee,
+  outOfHitbox: {
```

Comment:
I thought this already existed in outputs.. hm.

Reply:
Do you mean the "out of hitbox"? I didn't see anything that said that exactly.